generate.go

```diff
@@ -165,9 +165,9 @@ func generateConversion(cfg structConfig, t targetStruct, imports *imports) (gen
 		}
 
 		// the assignmentKind is <target> := <source> so target==LHS source==RHS
-		rawKind, ok := computeAssignment(field.Type(), sourceField.SourceType)
-		if !ok {
-			assignErrFn(nil)
+		rawKind, err := computeAssignment(field.Type(), sourceField.SourceType, imports)
```

Comment:
I really should've had this return an error instead of a boolean the first time, so i fixed that since i found it useful to return a real error in the case where you are using protobufs but not in the exact way this PR handles.